example/main.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+"""
+Documentation
+
+See also https://www.python-boilerplate.com/flask
+"""
+import os
+
+from flask import Flask, jsonify
+from flask_cors import CORS
+
+from featurehub_sdk.featurehub_config import FeatureHubConfig
+
+
+def create_app(config=None):
+    app = Flask(__name__)
+
+    # See http://flask.pocoo.org/docs/latest/config/
+    app.config.update(dict(DEBUG=True))
+    app.config.update(config or {})
+
+    # Setup cors headers to allow all domains
+    # https://flask-cors.readthedocs.io/en/latest/
+    CORS(app)
+
+    # Definition of the routes. Put them into their own file. See also
+    # Flask Blueprints: http://flask.pocoo.org/docs/latest/blueprints
+
+
+    # Create FeatureHub configuration
+    edge_url = 'https://zjbisc.demo.featurehub.io/'
```

Comment:
ideally these should come from an external config file, and we should be using our sample database for a consistent url and eval key, but this is v0.1